streams-activitypub-api/src/main/jsonschema/org/apache/streams/activitypub/api/pojo/WebfingerQueryRequest.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "http://json-schema.org/draft-07/schema",
   "id": "#",
   "type": "object",
-  "javaType": "org.apache.streams.activitypub.api.pojo.WebfingerQueryRequest",
+  "javaType": "org.apache.streams.activitypub.api.pojo.webfinger.WebfingerQueryRequest",
   "javaInterfaces": [
```

Comment:
there will be many of these endpoints so starting pattern of subdividing config/request/response pojos, api traits, remotes, and servlets into sub-packages